sample_consensus/include/pcl/sample_consensus/rransac.h

```diff
@@ -70,7 +74,7 @@ namespace pcl
       using SampleConsensus<PointT>::inliers_;
       using SampleConsensus<PointT>::probability_;
 
-      /** \brief RANSAC (Randomized RAndom SAmple Consensus) main constructor
+      /** \brief RANSAC (Randomized RANdom SAmple Consensus) main constructor
```

Comment:
The acronyms don't match with the expansion

Reply:
Oh, good eye! I only noticed the "n", haha